index.html

```diff
@@ -5,8 +5,8 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <script src="lesson_4/customerForm.js" defer></script>
-  <title>home work 8</title>
+  <script src="lesson_5/difficultCycles.js"></script>
```

Comment:
This js insertion will block the render((((

Reply:
I should put defer?